ros_ws/src/autonomous/src/emergency_stop.cpp

```diff
@@ -10,32 +10,23 @@ EmergencyStop::EmergencyStop()
 bool EmergencyStop::emergencyStop(const sensor_msgs::LaserScan::ConstPtr& lidar)
 {
     int available_samples = (lidar->angle_max - lidar->angle_min) / lidar->angle_increment;
```

Comment:
Because `lidar->ranges` is a `std::vector` this calculation could be replaced by simply calling the `.size()` function of `lidar->ranges`
```suggestion
    size_t available_samples = lidar->ranges.size();
```